ckanext/scheming/plugins.py

```diff
@@ -162,6 +162,7 @@ def group_types(self):
         return list(self._schemas)
 
     def setup_template_variables(self, context, data_dict, group_type=None):
```

Comment:
can we remove group_type from the signature?

Reply:
done